amd_openvx_extensions/amd_nn/src/topk_layer.cpp

```diff
@@ -0,0 +1,283 @@
+#include "kernels.h"
+
+
+static vx_status VX_CALLBACK validate(vx_node node, const vx_reference *parameters, vx_uint32 num, vx_meta_format metas[])

```

Comment:
add initialize and uninitialize function and keep the memory allocation in the local data of the node